tutorials/pretrain_redpajama.md

```diff
@@ -0,0 +1,173 @@
+# Pretrain Llama 2 on RedPajama
+
+This tutorial will walk you through setting up the RedPajama dataset and launching the pretraining script.
+
+## What's RedPajama
+
+[RedPajama](https://github.com/togethercomputer/RedPajama-Data) is an open-source reproduction of the original LLaMA training dataset.
+
+It contains a total of 1.2 trillion tokens, divided into
+
+```text
+Commoncrawl   878B
+C4            175B
+GitHub         59B
+Books          26B
+ArXiv          28B
+Wikipedia      24B
+StackExchange  20B
+```
```

Comment:
```suggestion
| Dataset       | Size |
|---------------|------|
| Commoncrawl   | 878B |
| C4            | 175B |
| GitHub        | 59B  |
| Books         | 26B  |
| ArXiv         | 28B  |
| Wikipedia     | 24B  |
| StackExchange | 20B  |
```

I have had opened online markdown editor anyway so here you go.

Reply:
thanks!